lib/npm_helpers.js

```diff
@@ -21,6 +21,6 @@ module.exports.rsplit = (string, character) => {
   return ['', string];
 }
 
-module.exports.getCacheFilename = (name, version) => {
-  return `${name}-${version}-registry.npmjs.org.tgz`;
+module.exports.getCacheFilename = () => {
+  return `nodejs-%{npm_name}-%{version}-registry.npmjs.org.tgz`;
```

Comment:
Doesn't this break because `bin/npm2rpm.js` uses it on the command line without RPM macros? It'd also like to avoid directly using the NPM name since that can include `@` and I'm not sure if that has a special meaning somewhere.